tests/ExceptionHandlerTest.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+namespace Fuzz\ImageResizer\Tests;
+
+use Fuzz\ImageResizer\ExceptionHandler;
+use InvalidArgumentException;
+use LogicException;
+use Mockery;
+use RuntimeException;
+
+class ExceptionHandlerTest extends ImageResizerTestCase
+{
+	/**
+	 * @runInSeparateProcess
+	 */
+	public function testItReturnsNiceErorOnInvalidArgumentException()
+	{
+		$exception = new InvalidArgumentException('Some message.');
+
+		$handled = ExceptionHandler::handleException($exception);
+
+		$this->assertEquals([
+			'error'   => 'bad_request',
+			'message' => $exception->getMessage(),
+			'trace'   => $exception->getTraceAsString(),
+		], $handled);
+	}
+
+	public function testItReturnsNiceErorOnLogicException()
+	{
+		$exception = new LogicException('Some message.');
+
+		$handled = ExceptionHandler::handleException($exception);
+
+		$this->assertEquals([
+			'error'   => 'bad_request',
+			'message' => $exception->getMessage(),
+			'trace'   => $exception->getTraceAsString(),
+		], $handled);
+	}
+
+	public function testItReturnsErrorUnknownIfInProduction()
```

Comment:
Last nitpicky naming comment, of which I could continue leaving indefinitely:

consider renaming to `testProductionReturnsUnknownErrors`.